docs/docs/20-quickstart.mdx

```diff
@@ -1,11 +1,12 @@
 ---
-description: Kargo Quickstart
+description: Find out how to quickstart Kargo and progress changes through multiple stages byinteracting with your GitOps repository and Argo CD `Application` resources
```

Comment:
```suggestion
description: Find out how to quickstart Kargo and progress changes through multiple stages byinteracting with your GitOps repository and Argo CD Application resources
```

I don't think the backticks make sense within metadata. I cannot speak to if or how they will affect SEO, but I'm pretty sure anything that might display this metadata is going to show the backticks instead of rendering the word within in a monospaced font.